$NetBSD$

Upstream commit c06bc0c449d0a988f2848be8ae2c4779dc6f26d0
conversation does not require libextractor or libgnunetfs


--- configure.ac.orig
+++ configure.ac
@@ -448,7 +448,7 @@ AS_IF([test "x$gnunet" = "x0"],
 AC_SUBST(GNUNET_CFLAGS)
 AC_SUBST(GNUNET_LIBS)
 
-AM_CONDITIONAL(HAVE_CONVERSATION, test x$gnunet_conversation != x0 -a "x$extractor" == "x1")
+AM_CONDITIONAL(HAVE_CONVERSATION, test x$gnunet_conversation != x0)
 
 backup_CPPFLAGS="$CPPFLAGS"
 CPPFLAGS="$GNUNET_CPPFLAGS $backup_CPPFLAGS"

--- src/conversation/Makefile.am.orig
+++ src/conversation/Makefile.am
@@ -38,7 +38,7 @@ gnunet_conversation_gtk_LDADD = \
   $(top_builddir)/src/lib/libgnunetgtk.la \
   @GTK_LIBS@ \
   @GLADE_LIBS@ @GNUNET_LIBS@ \
-  -lgnunetutil -lgnunetidentity -lgnunetconversation -lgnunetspeaker -lgnunetmicrophone -lgnunetnamestore -lgnunetgnsrecord -lgnunetgns -lgnunetfs -lextractor \
+  -lgnunetutil -lgnunetidentity -lgnunetconversation -lgnunetspeaker -lgnunetmicrophone -lgnunetnamestore -lgnunetgnsrecord -lgnunetgns \
   $(INTLLIBS)
 gnunet_conversation_gtk_LDFLAGS = \
   -export-dynamic
